sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -727,9 +727,9 @@ void stabilization_indi_rate_run(struct FloatRates rate_sp, bool in_flight)
   }
 
   // Set the stab_cmd to 42 to indicate that it is not used
-  stabilization_cmd[COMMAND_ROLL] = 42;
-  stabilization_cmd[COMMAND_PITCH] = 42;
-  stabilization_cmd[COMMAND_YAW] = 42;
+#if defined(COMMAND_ROLL) || defined(COMMAND_PITCH) || defined(COMMAND_YAW) 
+#warning "COMMAND_ROLL, PITCH, YAW not used in INDI: please remove it from your airframe file"
+#endif
```

Comment:
I'm fine with merging this even for next release, except maybe for these lines that will generate lots of warnings unless we update many airframe files. Can you keep it for later or just remove it ?

Reply:
sure...